src/shared/components/palette/organisation/write-organisation-field.component.ts

```diff
@@ -30,7 +33,7 @@ export class WriteOrganisationFieldComponent extends AbstractFieldWriteComponent
     super();
   }
 
-  ngOnInit() {
+  public ngOnInit() {
```

Comment:
what is the reason for adding the accessor on the ngOnInit()? Usually, we don't put that on this method.

Reply:
ts warning